.gitlab-ci.yml

```diff
@@ -448,19 +448,19 @@ dockerize:dockerExtensions:
     # Helm upgrade
     - helm init --upgrade
   script:
-    - helm upgrade $CI_COMMIT_REF_SLUG deploy/helm/magda --install --recreate-pods --namespace $CI_COMMIT_REF_SLUG -f deploy/helm/preview.yml --set global.image.repository=registry.gitlab.com/magda-data/magda/data61,global.image.tag=$CI_COMMIT_REF_SLUG,ingress.hostname=$CI_COMMIT_REF_SLUG.dev.magda.io,correspondence-api.externalUrl=http://$CI_COMMIT_REF_SLUG.dev.magda.io --timeout 1200 --wait
+    - helm upgrade $CI_COMMIT_REF_SLUG deploy/helm/magda --install --recreate-pods --namespace $CI_COMMIT_REF_SLUG -f deploy/helm/preview.yml --set global.image.repository=registry.gitlab.com/magda-data/magda/data61,global.image.tag=$CI_COMMIT_REF_SLUG,ingress.hostname=$CI_COMMIT_REF_SLUG.dev.magda.io,correspondence-api.externalUrl=https://$CI_COMMIT_REF_SLUG.dev.magda.io --timeout 1200 --wait
```

Comment:
Sorry. I really should check everything in one go.
Should be this line:
```
correspondence-api.externalUrl=https://$CI_COMMIT_REF_SLUG.dev.magda.io
```
or this line at the bottom:
```
web-server.externalUrl
```
both be:
```
global.externalUrl=https://$CI_COMMIT_REF_SLUG.dev.magda.io
```
currently, the value is still return localhost in server-config.js

Reply:
I actually not sure the outcome of 
```
"--baseExternalUrl", {{ .Values.externalUrl | default .Values.global.externalUrl | quote }}
```
So please correct me if I suggest something wrongly...

Reply:
Yeah so the idea is that I override the externalUrl but just for the web-server pod, so that the link to nationalmap works.

But now I look at it with fresh eyes it doesn't make much sense, the real solution is to make it so that the nationalmap link goes to `baseUri` instead of `externalUri` because it'll be pointing at the API.

Also I'm not sure why `correspondence-api` needs its own external uri, I'll look at that too.